src/styles/header.scss

```diff
@@ -19,6 +20,7 @@
 
 	/* logo */
 	#header,
+	#header-beta .header-logo,
```

Comment:
header_beta

Reply:
Actually, not even sure we need this. As far as i can tell the `#logo_beta svg` takes care of any cases this would.

Reply:
And I think I made a mistake when converting from `css` to `scss`, this should probably be just `.header-logo` anyway.